ethcore/sync/src/blocks.rs

```diff
@@ -317,42 +317,41 @@ impl BlockCollection {
 			return Vec::new();
 		}
 
-		let mut drained = Vec::new();
+		// todo[dvdplm]: should be possible to know how many these are.
```

Comment:
if there is no way, let's remove the todo

Reply:
It's actually not trivial to find out the size of these collections, likely we keep blocks from all kinds of forks in the `self.blocks`/`self.parents` collections and there's no good way to anticipate how many blocks we're going to drain. Most of the time we end up draining 0 blocks and then we drain a whole bunch of them, up to 30 000. Not sure why this is tbh.